glue/core/data.py

```diff
@@ -1828,9 +1831,16 @@ def compute_statistic(self, statistic, cid, subset_state=None, axis=None,
             # of the if statement above.
```

Comment:
```suggestion
            # of the if statement above. Likewise if a view was specified,
            # only the result within the view is returned.
```
the `view` parameter could need some real documentation in general...